R/InstanceSettings.R

```diff
@@ -37,13 +37,13 @@ InstanceSettings <- R6::R6Class( # nolint object_name_linter
         "db_user_password", # api
         "lamindb_version" # api
       )
-      missing_column <- setdiff(expected_keys, names(settings))
-      if (length(missing_column) > 0) {
-        cli_abort("Missing column: ", missing_column)
+      missing_keys <- setdiff(expected_keys, names(settings))
+      if (length(missing_key) > 0) {
```

Comment:
Missing a `s` here

```suggestion
if (length(missing_keys) > 0) {
```

Reply:
Would accept this suggestion but already fixed it myself based on the feedback you gave